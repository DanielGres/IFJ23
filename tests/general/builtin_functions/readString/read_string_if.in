// DOBRE
let a: String? = readString()
if let a {
    write("The value of 'a' is: ", a)
} else {
    write("Error reading input");
}