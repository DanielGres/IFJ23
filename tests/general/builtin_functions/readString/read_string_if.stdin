dannyho_kava_preti_good