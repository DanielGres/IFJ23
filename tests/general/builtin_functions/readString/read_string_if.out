The value of 'a' is: dannyho_kava_preti_good